Gateway/Http/Client/Payment.php

```diff
@@ -50,10 +48,19 @@ public function __construct(
      */
     public function placeRequest(TransferInterface $transferObject)
     {
+
         $this->omise->defineUserAgent();
         $this->omise->defineApiVersion();
         $this->omise->defineApiKeys();
+        
+        $transferObjectBody = $transferObject->getBody();
+
+        // if charge_id already exists than action is 'manual capture'
+        if (isset($transferObjectBody['charge_id'])) {
+            $charge = $this->apiCharge->find($transferObjectBody['charge_id']);
+            return ['charge' => $charge->capture()];
+        }
 
-        return ['charge' => $this->apiCharge->create($transferObject->getBody())];
+        return ['charge' => $this->apiCharge->create($transferObjectBody)];
```

Comment:
In this function, `'charge'` has been hardcoded whereas in the file below a constant has been used. Should we be consistent?